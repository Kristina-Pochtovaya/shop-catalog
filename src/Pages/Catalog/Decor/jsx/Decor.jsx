import React from 'react';
import {Header} from '../../../Main/jsx/Header';
import decor1 from '../img/decor1.jpg';
import decor2 from '../img/decor2.jpg';
import decor3 from '../img/decor3.jpg';

export const Decor = () => (
  <>
    <Header linkItem={<button type="button" className="buttonBack">Назад</button>} link="/main-page" />
    <div className="container">
      <div className="items-wrap">
        <div className="item">
          <img src={decor1} alt="Часы настенные" title="Часы настенные" />
          <p>33 РУБ.</p>
          <h4>Часы настенные</h4>
        </div>
        <div className="item">
          <img src={decor2} alt="Репродукция в раме" title="Репродукция в раме" />
          <p>42 РУБ.</p>
          <h4>Репродукция в раме</h4>
        </div>
        <div className="item">
          <img src={decor3} alt="Подушка декоративная" title="Подушка декоративная" />
          <p>28 РУБ.</p>
          <h4>Подушка декоративная</h4>
        </div>
      </div>
    </div>
  </>
);



