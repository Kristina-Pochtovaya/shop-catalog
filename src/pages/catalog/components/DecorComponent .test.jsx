import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Decor from './DecorComponent';

configure({ adapter: new Adapter() });

test('should render an Decor', () => {
  const decor = shallow(<Decor />);
  expect(toJson(decor)).toMatchSnapshot();
});
