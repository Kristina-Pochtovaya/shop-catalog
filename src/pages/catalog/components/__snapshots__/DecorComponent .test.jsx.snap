// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an Decor 1`] = `
<Fragment>
  <Header
    disabled={false}
    link="/main-page"
    linkItem={
      <button
        className="buttonBack"
        type="button"
      >
        Назад
      </button>
    }
  />
  <div
    className="container-wrap"
  >
    <div
      className="items"
    >
      <Connect(CatalogItem)
        counter={1}
        description="Часы настенные Troyka 91900929 (230)"
        id={9}
        img={
          <img
            alt="Часы настенные Troyka 91900929 (230)"
            className="imgItem"
            src="decor1.jpg"
            title="Часы настенные Troyka 91900929 (230)"
          />
        }
        inStock={true}
        price={33}
        setPopupBasketctive={[Function]}
      />
      <Connect(CatalogItem)
        counter={1}
        description="Репродукция в раме Styler Осень-1 (OB-02591) 500x500"
        id={10}
        img={
          <img
            alt="Репродукция в раме Styler Осень-1 (OB-02591) 500x500"
            className="imgItem"
            src="decor2.jpg"
            title="Репродукция в раме Styler Осень-1 (OB-02591) 500x500"
          />
        }
        inStock={true}
        price={42}
        setPopupBasketctive={[Function]}
      />
      <Connect(CatalogItem)
        counter={1}
        description="Подушка декоративная Nivasan Дино"
        id={11}
        img={
          <img
            alt="Подушка декоративная Nivasan Дино"
            className="imgItem"
            src="decor3.jpg"
            title="Подушка декоративная Nivasan Дино"
          />
        }
        inStock={true}
        price={28}
        setPopupBasketctive={[Function]}
      />
    </div>
  </div>
  <PopUp
    active={false}
    setActive={[Function]}
  >
    <PopupBasket
      closePopup={[Function]}
    />
  </PopUp>
  <Footer />
</Fragment>
`;
